doc/flame_bloc/bloc.md

```diff
@@ -0,0 +1,134 @@
+# flame_bloc
+
+`flame_bloc` is a bridge library for using Bloc in your Flame game. `flame_bloc` offers a simple and
+natural (as in similar to flutter_bloc) way to use blocs and cubits inside a FlameGame. Bloc
+attempts to make game state changes predictable by regulating when a game state change can occur and
+enforcing a single way to change game state throughout an entire Game.
```

Comment:
I don't think this is true, as well as in flutter, you can mix blocs with value change notifiers or setState, in flame you can use flame bloc alongside also managing your state inside your components or any other approach.

So here I would change to sound more like it offers you another way of managing your state instead of enforcing.

Makes sense?

Reply:
makes  👍